src/domain.rs

```diff
@@ -398,7 +414,16 @@ impl<F> Domain<F> {
             if rcount != 0 {
                 self.count.fetch_add(rcount, Ordering::Release);
             }
-            rcount = self.check_count_threshold();
+            #[cfg(feature = "std")]
```

Comment:
Is there a better way to do this? Without the scope rustc complains that It can't remove an expression in this position.